torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -1594,7 +1588,7 @@ struct PythonPrintImpl {
   bool enforce_importable_;
 
   // The least version that supports all printed ops
-  uint64_t min_version_ = 0;
+  uint64_t min_version_ = caffe2::serialize::kProducedFileFormatVersion;
```

Comment:
This is no longer `min_version`, instead it is the exact version I think. Maybe we don't even need to keep track of `min_version` as a member of this class, instead we can simply write out `caffe2::serialize::kProducedFileFormatVersion` to the `version` file.

Reply:
done